mapdata/WarcraftLegacies/UpgradeData/Skin/1145193554.json

```diff
@@ -0,0 +1,53 @@
+{
+  "OldId": 1702062162,
+  "NewId": 1145193554,
+  "Unk": [
+    0
+  ],
+  "Modifications": [
+    {
+      "Level": 1,
+      "Id": 829579623,
+      "Type": 3,
+      "Value": "ReplaceableTextures\\CommandButtons\\BTNCryptFiendBurrow.blp"
```

Comment:
I think we need to give this ability a custom icon. It's doubly important here because it _sounds_ like it might just be the same as Burrow, but it's actually very different.